### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by hungchihsueh** [vercel/community#4117] [Account Issue] [answered]
### Question

My account have been block recently for unknown reason.
Since I am unable to login with my githjub ,I can't reach out  to the support team.
i dont do any bussiness work on my account, and i dont get any email about it.
if anyone on vercel support team see this please contact me , i just want to get my content and domain back thank you

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps.

For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)

To open a ticket please use the appropriate channel found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
